src/styles/index.scss

```diff
@@ -1,3 +1,100 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import './blocks/card';
+@import './blocks/stars';
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+
+  --col-number: 1;
+}
+
+header {
+  background-color: $color-bacground;
```

Comment:
There is a typo in the variable name `$color-bacground`. It should be `$color-background` to maintain consistency with the naming convention.